static-site/components/nav/nav-logo.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import Image from "next/image";
+import { usePathname } from "next/navigation";
+import React from "react";
+
+import { groupsApp, siteTitle } from "../../data/BaseConfig";
+import logo from "../../static/logos/nextstrain-logo-small.png";
+
+export default function NavLogo(): React.ReactElement | null {
+  const pathname = usePathname();
+
+  return pathname !== "/" ? (
+    <>
+      <Logo />
+      <LogoType />
+    </>
+  ) : null;
+}
+
+const Logo = (): React.ReactElement => (
+  <a href="/" className="logo">
+    <Image src={logo} alt="Nextstrain (logo)" width="40" />
+  </a>
+);
+
+const LogoType = (): React.ReactElement => {
+  const rainbowTitle = siteTitle
+    .split("")
+    .map((letter, i) => (
+      <span key={i} style={{ color: `var(--titleColor${i})` }}>{letter}</span>
+    ));
+  const SubTitle = (): React.ReactElement => (
+    <div className="wordmark-subtitle">Groups Server</div>
+  );
+
+  return (
+    <a href="/" className="wordmark">
```

Comment:
Looks like an oversight; I can look at converting it over. 

I will say, this will make the resulting CSS uglier, because the `nav` selector won't be allowed in a CSS Module, because it is not "pure".

Reply:
Couldn't the `nav` selector be replaced with `.nav` and used with `<nav className={styles.nav}>`?

Reply:
> Couldn't the `nav` selector be replaced with `.nav` and used with `<nav className={styles.nav}>`?

I mean, yeah, that's what I'm planning on doing, but that ☝️ is IMO uglier than `<nav>`, and the CSS will change from the clean and nested stuff that's there now to having to prepend `.nav` to all other selectors. 

I'm not going to change it at this point but I have mild regrets about going the CSS Modules route; the bad parts > the good, IMO.